R/zzz.R

```diff
@@ -2,6 +2,11 @@
   run_on_load()
   ns <- ns_env(pkgname)
 
+  # Force reload global shims if developing pkgload itself
+  if (is_loading()) {
+    insert_global_shims(TRUE)
```

Comment:
Self-documenting named argument:

```suggestion
    insert_global_shims(force = TRUE)
```